man/tpm2-totp.1.md

```diff
@@ -3,151 +3,173 @@
 
-  * `-N <nvindex>`, `--nvindex <nvindex>`:
-    TPM NV index to store data (default: 0x018094AF)
+  * **-N** *\<nvindex\>*, **--nvindex** *\<nvindex\>*:
+    TPM NV index to store data (default: *0x018094AF*)
 
-  * `-p <pcr>[,<pcr>[,...]]`, `--pcrs <pcr>[,<pcr>[,...]]`:
-    Selected PCR registers (default: 0,2,4,6)
+  * **-p** *\<pcr\>*[*,\<pcr\>*[*,...*]], **--pcrs** *\<pcr\>*[*,\<pcr\>*[*,...*]]:
+    Selected PCR registers (default: *0,2,4,6*)
 
-  * `-P <password>`, `--password <password>`:
-    Password for the secret (default: none) (commands: init, recover, reseal)
-    Read from stdin if `-` (recommended).
-    Must not contain `\0`.
+  * **-P** *\<password\>*, **--password** *\<password\>*:
+    Password for the secret (default: *none*) (commands: **init,** **recover,** **reseal)**.
+    Read from stdin if *-* (recommended).
+    Must not contain *\0*.
 
-  * `-t`, `--time`:
-    Display the date/time of the TOTP calculation (commands: show)
+  * **-t**, **--time**:
+    Display the date/time of the TOTP calculation (commands: **show**)
 
-  * `-T <tcti-name>[:<tcti-config>]`, `--tcti <tcti-name>[:<tcti-config>]`:
+  * **-T** *\<tcti-name\>*[*:\<tcti-config\>*], **--tcti** *\<tcti-name\>*[*:\<tcti-config\>*]:
     Select the TCTI to use. *tcti-name* is the name of the TCTI library.
     If present, the configuration string *tcti-config* is passed verbatim to the
     chosen TCTI library.
 
     The TCTI can additionally be specified using the environment variable
-    `TPM2TOTP_TCTI`. If both the command line option and the environment
+    **TPM2TOTP_TCTI**. If both the command line option and the environment
     variable are present, the command line option is used.
 
     If no TCTI is specified, the default TCTI configured on the system is used.
 
-  * `-v`, `--verbose`:
+  * **-v**, **--verbose**:
     Print verbose messages
 
 # EXAMPLES
 
 ## Setup
 The TOTP secret can be initialized with and without password. It is recommended to
-set a password `-P` in order to enable recovery options. Further, it is strongly
-recommended to provide the password via stdin, rather than directly as a
+set a password **-P** in order to enable recovery options. Further, it is strongly
+recommended to provide the password via *stdin*, rather than directly as a
 command line option, to protect it from other processes, shell history, etc.
-Also the PCRs and PCR banks can be selected `-p` and `-b`. Default values are
-PCRs `0,2,4` and banks `SHA1, SHA256`.
-```
-tpm2-totp init
-
-tpm2-totp -P - init
-verysecret<CTRL-D>
-# or (recommended)
-gpg --decrypt /path/to/password.gpg | tpm2-totp -P - init
-# or (discouraged)
-tpm2-totp -P verysecret init
-
-tpm2-totp -P - -p 0,1,2,3,4,5,6 init
-tpm2-totp -p 0,1,2,3,4,5,6 -b SHA1,SHA256 init
-```
+Also the PCRs and PCR banks can be selected **-p** and **-b**. Default values
+are PCRs *0,2,4* and banks *SHA1,SHA256*.
+
+**tpm2-totp init**
+
+
+**tpm2-totp -P** *-* **init**  
+\> *verysecret***\<CTRL-D\>**
```

Comment:
> The whitespace checker fail is anticipated - the two trailing whitespaces are necessary to coerce pandoc into rendering sensible man pages. <br> doesn't cut it here.

I'd prefer to use a [backslash, followed directly like a line break](https://pandoc.org/MANUAL.html#extension-escaped_line_breaks) instead of the trailing whitespace to be able to keep the whitespace checker without modification, otherwise LGTM!

```markdown
**tpm2-totp -P** *-* **init**\
\> *verysecret***\<CTRL-D\>**
```